lib/trailblazer/rails/railtie.rb

```diff
@@ -3,8 +3,37 @@
 
 module Trailblazer
   class Railtie < ::Rails::Railtie
+    def self.load_root(root)
+
+      # Loader has no concept for base dir, must chdir so that it can find files local to the root
+      Dir.chdir(root) do
+        # For STI models, we need to sort all models before concepts, gather files first from loader
+        files = []
+        Loader.new.({debug: false, insert: [ModelFile, before: Loader::AddConceptFiles]}) { |file|
```

Comment:
@rosskevin I think "blindly" loading the models first would resolve most problems, but we should offer that as an alternative loading strategy. I will take a stab at your code and rearrange it a bit. 

@aiomaster I hear you, but that would result in even more work for me since some semantics will definitely change when we "optimize" the code.